src/hooks/useStableSwapEstimateRemoveLiquidity.ts

```diff
@@ -58,6 +60,10 @@ export default function useStableSwapEstimateRemoveLiquidity({
   const getEstimatedAmounts = useCallback(async () => {
     setError(null)
 
+    if (amount?.greaterThan(lpTokenBalance ?? BIG_INT_ZERO)) {
+      setError({ ...new Error(), reason: 'Insufficient LP amount' })
```

Comment:
I think this function should only handle estimations, instead of also validating user inputs